packages/opentelemetry-api-metrics/src/NoopMeter.ts

```diff
@@ -33,6 +33,7 @@ import {
   BoundBaseObserver,
 } from './types/BoundInstrument';
 import { ObserverResult } from './types/ObserverResult';
+import { Observation } from '.';
```

Comment:
This can help reduce the risk of circular dependencies in the future

```suggestion
import { Observation } from './types/Observation';
```